src/functions/uploadToRSIS/framework/repo/rsis-repository.ts

```diff
@@ -27,7 +26,7 @@ export const saveTestResult = async (
   const params = mappedFields.map(field => field.val);
 
   if (config.useRSIS) {
-    if (isUndefined(connection)) {
+    if (connection === undefined) {
```

Comment:
why the removal of the util function?

Reply:
https://github.com/nodejs/node-v0.x-archive/issues/8235
The method has been deprecated